generators/app/index.js

```diff
@@ -49,6 +49,7 @@ module.exports = Generator.extend({
 
   default: function () {
     var cmd = [
+      'rm -f master.tar.gz pattern-lab-starter-master patternlab',
```

Comment:
Ah, I thought that previous comment was a request to add another try block. I haven't encountered what you're talking about yet so I'm inclined to wait for a follow-up on that bit. Unless you have more specifics?